drivers/sensor/opt3001/opt3001.c

```diff
@@ -61,6 +62,50 @@ static int opt3001_reg_update(const struct device *dev, uint8_t reg,
 	return opt3001_reg_write(dev, reg, new_val);
 }
 
+#if defined(CONFIG_OPT3001_TRIGGER)
+static int opt3001_attr_set_low_th(const struct device *dev,
+				   const struct sensor_value *value)
+{
+	return opt3001_set_th(dev, value->val1, 0x02);
+}
+
+static int opt3001_attr_set_high_th(const struct device *dev,
+				    const struct sensor_value *value)
+{
+	return opt3001_set_th(dev, value->val1, 0x03);
+}
+
+static int opt3001_attr_set_light(const struct device *dev,
```

Comment:
Can this be renamed to `opt3001_attr_set_threshold`?

Reply:
Renaming done.